wcfsetup/install/files/acp/update_5.4_checkOwnerGroup.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use wcf\data\user\group\UserGroup;
+
+if (UserGroup::getOwnerGroupID() === null) {
+	throw new \Exception('No owner group');
```

Comment:
This should produce a meaningful error, similar to this:
https://github.com/WoltLab/WCF/blob/490da6f037fdf904c78c124786665eddbd491ee5/wcfsetup/install/files/acp/update_com.woltlab.wcf_5.3_preUpdate.php#L25-L32